controllers/routes/routes.js

```diff
@@ -92,45 +92,63 @@ var appRouter = function(app) {
                 });
             }
         })
-        .get("/buy", function(req, res) {
-            // buy random product
+        .post("/orders", function(req, res) {
```

Comment:
So the new route to buy a product is a POST to orders? That does not sound very clear to me.

Any other ideas?